docs/src/pages/[platform]/getting-started/usage/nextjs/react.mdx

```diff
@@ -24,10 +24,239 @@ Then, install the Amplify UI React package:
 
 <InstallScripts framework="react" />
 
+### Usage
+
+<Tabs>
+  <TabItem title="Next.js 13.4+">
+  <Alert variation="info" role="none">
+    Amplify UI components only work in Client Components. If you need to load them into a Server Component, you need to convert them into client-side components by adding a `'use client'` at the top of your file. Please see instructions below on how you could achieve this. For more info, visit Next.js [documentation](https://nextjs.org/docs/getting-started/react-essentials#third-party-packages).
+  </Alert>
+#### UI Components
+
+##### Single Component
+
+To use a Amplify UI component in Server Components, you must re-export them as Client Components.
+
+An example below is shown for using Amplify UI `Button` component:
+
+First, create a Client Component that exports the Amplify component.  For example, create`AmplifyUIComponents.jsx` placing `'use client'` at the top , import the component and styles and re-export `Button` component.
+
+    <ExampleCode>
+
+    ```tsx
+    // AmplifyUIComponents.jsx
+    'use client';
+
+    import { Button } from '@aws-amplify/ui-react';
+    import '@aws-amplify/ui-react/styles.css';
+
+    export { Button };
+    ```
+
+    </ExampleCode>
+
+Then, import `Button` from the Client Component and use it anywhere you want, e.g., `app/page.jsx`.
+
+    <ExampleCode>
+
+    ```tsx
+    // app/page.jsx
+    import { Button } from './AmplifyUIComponents';
+
+    export default function Home() {
+      return <Button>Hello, Amplify!</Button>;
+    }
+    ```
+
+    </ExampleCode>
+
+##### Multiple Components
+
+To use more Amplify UI components, you need to re-export them in the same Client Component `AmplifyUIComponents.jsx` and then load them from there.
+
+    <ExampleCode>
+
+    ```tsx
+    // AmplifyUIComponents.jsx
+    'use client';
+
+    import { Alert, Button, Card } from '@aws-amplify/ui-react';
+    import '@aws-amplify/ui-react/styles.css';
+
+    export { Alert, Button, Card };
+    ```
+
+    </ExampleCode>
+
+#### Connected Components
+    To use a Amplify UI Connected Component in app directory, you must set up `Amplify.configure` in a Client Component.
+
+    First, create a Client Component, for example, `configure.js` by placing `'use client'` at the top of the file under `app` directory:
+
+    <ExampleCode>
+
+    ```tsx
+    // configure.js
+
```

Comment:
```suggestion
'use client';
```

Reply:
Thinking we should not limit the separate config to client. It could be running on the server as well. Wouldn't that give users an impression that it is client required only?